8.1/Dockerfile

```diff
@@ -66,11 +66,9 @@ RUN mkdir -p jre-tmp
 RUN mkdir -p /var/lib/ignition && \
     mv data /var/lib/ignition/ && \
     mv user-lib /var/lib/ignition/ && \
-    mv temp /var/lib/ignition/data && \
     mv logs /var/log/ignition && \
     ln -s /var/lib/ignition/data data && \
     ln -s /var/lib/ignition/user-lib user-lib && \
-    ln -s /var/lib/ignition/data/temp temp && \
```

Comment:
No longer need these since an upstream fix now correctly accounts for the `temp` folder being on an alternate filesystem (e.g. volume mount in Docker)